### ISSUE TITLE
Restore "Browse Modules" Section

### ISSUE THREAD

**Issue by bretonics** [bioperl/bioperl.github.io#37]
The [old browser page](http://bioperl.org/wiki/Category:Core_Modules) `http://bioperl.org/wiki/Category:Core_Modules` for all core modules was a great resource!

It should be restored in the new website.

**Comment by cjfields** [comment]
@bretonics it's feasible that we can set this up, but it may be easier to link out to the relevant documentation on a more reliably maintainable site like MetaCPAN or readthedocs.

**Comment by bretonics** [comment]
Agreed, it would probably be best to link out to those repos, but having that list of all core modules was a great first-hand resource.

Thanks 👍🏻

**Comment by bretonics** [comment]
@cjfields, any plans whether or when this will be brought back? It's much harder finding all the available modules when looking for what's available. 

I always eventually go to CPAN, but searching for an exact match is different than looking over a list (say you want to look for modules never used before with features I might need). Can't seem to find an alternative and I know I've seen modules on that list before that I need now, but can't remember the names, but recall the features.

**Comment by cjfields** [comment]
@bretonics There are plans to do this, but it may require thinking about the use case here.  My suggestion is that we open a conversation on the mail list to see how this could be implemented on the current web site and whether we should use available resources effectively (which would be the best option) or come up with our own pages.  

To be honest, this is one of the reasons I like the idea of maybe using ReadTheDocs for this, as we can tie the module documentation to versions.  We mainly need people who can work on implementing this faster,and this would be the perfect place to jump in and help if one were so inclined _hint hint_ 😄

**Comment by bretonics** [comment]
Yes, definitely! I like the idea of using ReadTheDocs too. Do you happen to have the HTML files from the wiki you had before listing the modules and their pages?

**Comment by cjfields** [comment]
@bretonics we have tarballs of the wiki dumps, but it may be as simple as Perl POD support within ReadTheDocs:

rtfd/readthedocs.org#1319
